.travis.yml

```diff
@@ -12,7 +12,7 @@ install:
 script:
     - tox -e $TOX_ENV
     - ./node_modules/.bin/grunt --verbose
-    - flake8 kickoff/ --exclude=kickoff/test/ --max-line-length=2000 --ignore=E402
+    - flake8 kickoff/ --exclude=kickoff/test/ --max-line-length=2000 --ignore=E402,E711
```

Comment:
BTW, it has to be `filter(something != None)` to make sqlalchemy magic work (they override `__neq__()` AFAIK), `something is not None` is not the same.